contribute/release.md

```diff
@@ -7,17 +7,17 @@ To generate dashboards and alert rules for monitoring helm chart and release a n
 
 2. Update the dashboards and rules folder in [build.sh](../jsonnet/openebs-mixin/build.sh). The auto generated json files will be inside this folder.  
    
-   For example: If you add dashboards and alert rules for mayastor in openebs-mixin, then add mayastor as a folder name in build.sh file. 
+   For example: If you add dashboards and alert rules for Mayastor in openebs-mixin, then add mayastor as a folder name in build.sh file. 
    ```
 	generateDashboards(){
 		. . . 
-		dashboardsFolder=(cStor jiva localPV npd lvmLocalPV ndm mayastor)
+		dashboardsFolder=(npd lvmLocalPV zfsLocalPV mayastor)
```

Comment:
What is npd?

Reply:
npd -> node-problem-detector

with @avishnu we have this thought to change `openebs-monitoring` to `monitoring`. Should i update it ?

Reply:
Ya, I think we should change it to monitoring? Can you list down where all the changes would reflect?

Reply:
1. Chart name and we are using this name for prepending the resource names like following.

2. pod names after installing monitoring helm chart

```
aman@aman-mbp:charts$ ko get po | grep openebs-monitoring
alertmanager-openebs-monitoring-kube-pr-alertmanager-0   2/2     Running   0          4h34m
openebs-monitoring-grafana-fb9cc8cd4-l9sdx               3/3     Running   0          4h34m
openebs-monitoring-kube-pr-operator-bd7fdd6cd-vq5x8      1/1     Running   0          4h34m
openebs-monitoring-kube-state-metrics-c478cc764-9d7hm    1/1     Running   0          4h34m
openebs-monitoring-prometheus-node-exporter-64zvh        1/1     Running   0          4h34m
openebs-monitoring-prometheus-node-exporter-74wxb        1/1     Running   0          4h34m
openebs-monitoring-prometheus-node-exporter-g454f        1/1     Running   0          4h34m
openebs-monitoring-prometheus-node-exporter-ghndk        1/1     Running   0          4h34m
prometheus-openebs-monitoring-kube-pr-prometheus-0       2/2     Running   0          4h34m
```

3. service-monitor names

```
aman@aman-mbp:charts$ k get servicemonitor -n openebs
NAME                                                 AGE
openebs-monitoring-grafana                           4h36m
openebs-monitoring-kube-pr-alertmanager              4h36m
openebs-monitoring-kube-pr-apiserver                 4h36m
openebs-monitoring-kube-pr-coredns                   4h36m
openebs-monitoring-kube-pr-kube-controller-manager   4h36m
openebs-monitoring-kube-pr-kube-etcd                 4h36m
openebs-monitoring-kube-pr-kube-proxy                4h36m
openebs-monitoring-kube-pr-kube-scheduler            4h36m
openebs-monitoring-kube-pr-kubelet                   4h36m
openebs-monitoring-kube-pr-operator                  4h36m
openebs-monitoring-kube-pr-prometheus                4h36m
openebs-monitoring-kube-state-metrics                4h36m
openebs-monitoring-lvmlocalpv                        4h36m
openebs-monitoring-mayastor                          4h36m
openebs-monitoring-prometheus-node-exporter          4h36m
```

4. services
aman@aman-mbp:charts$ k get svc -n openebs | grep openebs-monitoring
```
openebs-monitoring-grafana                    NodePort    10.109.38.93     <none>        80:32515/TCP                                            4h37m
openebs-monitoring-kube-pr-alertmanager       NodePort    10.102.59.49     <none>        9093:30903/TCP,8080:31936/TCP                           4h37m
openebs-monitoring-kube-pr-operator           ClusterIP   10.105.103.10    <none>        443/TCP                                                 4h37m
openebs-monitoring-kube-pr-prometheus         NodePort    10.99.162.154    <none>        9090:32514/TCP,8080:31856/TCP                           4h37m
openebs-monitoring-kube-state-metrics         ClusterIP   10.105.215.88    <none>        8080/TCP                                                4h37m
openebs-monitoring-prometheus-node-exporter   ClusterIP   10.109.138.28    <none>        9100/TCP
```